### ISSUE TITLE
Integrate Answers (Analytics)

### ISSUE THREAD

**Issue by eugenkiss** [eugenkiss/chanobol#98]
See http://www.reddit.com/r/androiddev/comments/2uo0yi/google_analytics_for_android_or_flurry_what_do/coa75ty

**Comment by mitchross** [comment]
Super easy to add, just need you to make an account on Fabric. fabric.io/

Rest is super easy just follow instructions. Dont commit your crashlytics key to the repo!!

**Comment by eugenkiss** [comment]
Yeah, I know, I'm using Crashlytics already ;). It's just a reminder for me.
